back/src/middlewares/authenticate.ts

```diff
@@ -1,8 +1,8 @@
-import { NextFunction, Request, Response } from 'express';
+import { NextFunction, Request, Response, response } from 'express';
```

Comment:
What is that for ?

Reply:
Resolved, I did something wrong in the use of "res" variable